Makefile

```diff
@@ -1,3 +1,34 @@
+GITHUB_API_KEY ?=
+GITHUB_ORG ?=
+LOGS ?= $(PWD)/dev/logs
+ERRORS ?= $(PWD)/dev/errors
+INIT ?= $(PWD)/dev/init.d
+SHARED ?= $(PWD)/dev/shared
+
 build:
-	@mkdir -p bin; \
-	CGO_ENABLED=0 go build -o bin/docsrv docsrv.go
+	@mkdir -p bin;
+	@CGO_ENABLED=0 go build -o bin/docsrv docsrv.go
+
+docker_container_name = docsrv-instance
+develop: build drop-running-instances
+	@docker build -t docsrv .
+	docker run \
+		--name $(docker_container_name) \
+		--publish 9090:9090 \
+		--env GITHUB_API_KEY="$(GITHUB_API_KEY)" \
+		--env GITHUB_ORG="$(GITHUB_ORG)" \
+		--volume $(LOGS):/var/log/docsrv \
```

Comment:
FYI: in kubernetes deployment we're handling logging through standard output / standard error  with Docker logging driver. So your containers should output logs to standard error, not log files.
Anyway, I guess that docsrv might output additional log files of the build tools for debugging purposes.